src/styles/variables.less

```diff
@@ -124,7 +124,7 @@
 // Typography
 // Font, line-height, and color for body text, headings, and more.
 
-@font-family-base:      Apple-System, Arial, Helvetica, 'PingFang SC', 'Hiragino Sans GB', 'Microsoft YaHei', STXihei, sans-serif;
+@font-family-base:      -apple-system, Arial, Helvetica, 'PingFang SC', 'Hiragino Sans GB', 'Microsoft YaHei', STXihei, sans-serif;
```

Comment:
It is a way to be compatible with old browsers, defining [Apple-System](https://github.com/rsuite/rsuite/blob/main/src/styles/typography.less#L121
).
```css
// font-face
@font-face {
  font-family: Apple-System;
  src: local(-apple-system), local(BlinkMacSystemFont), local(system-ui);
}
```


I think it can be changed to:
```less
@font-family-base: -apple-system , BlinkMacSystemFont, ......;
```

Also need to remove the definition of `Apple-System`.

Reply:
None of `local(-apple-system)`, `local(BlinkMacSystemFont)`, `local(system-ui)` works in Chrome.